src/commands/node/install.ts

```diff
@@ -243,7 +243,7 @@ export default class InstallCommand extends BaseCommand {
 			},
 		]);
 
-		await tasks.run().catch(output => {
```

Comment:
If we do not use `await` then we need to return.
But I think try-catch is fine here